readthedocs/api/utils.py

```diff
@@ -154,7 +159,7 @@ def is_authenticated(self, request, **kwargs):
 
 
 class EnhancedModelResource(ModelResource):
-    def obj_get_list(self, request=None, *args, **kwargs):
+    def obj_get_list(self, request=None, **kwargs):
```

Comment:
Though not used here, we should also ensure `args` was not used or required. For instance, if `ModelResource` has additional positional arguments.